airflow/providers/dbt/cloud/sensors/dbt.py

```diff
@@ -49,13 +51,30 @@ def __init__(
         dbt_cloud_conn_id: str = DbtCloudHook.default_conn_name,
         run_id: int,
         account_id: int | None = None,
+        deferrable: bool = False,
         **kwargs,
     ) -> None:
+        if deferrable and "poke_interval" not in kwargs:
+            # TODO: Remove once deprecated
+            if "polling_interval" in kwargs:
+                kwargs["poke_interval"] = kwargs["polling_interval"]
+                warnings.warn(
+                    "Argument `poll_interval` is deprecated and will be removed "
+                    "in a future release.  Please use `poke_interval` instead.",
+                    DeprecationWarning,
+                    stacklevel=2,
+                )
+            else:
+                kwargs["timeout"] = 60 * 60 * 24 * 7
```

Comment:
Should there be a check to make sure `timeout` is not in `kwargs` as well _just in case_ users happen to specify it explicitly while running the sensor in deferrable mode?